app/src/main/java/com/rockthevote/grommet/ui/registration/RegistrationActivity.java

```diff
@@ -61,6 +69,8 @@ public RegistrationActivity() {
         LocaleUtils.updateConfig(this);
     }
 
+    private RegistrationViewModel viewModel;
```

Comment:
thought you would have already had to have this VM in this class?

Reply:
It was being scoped to the Activity from the Fragment, but the Activity itself didn't need it. I think `BaseRegistrationFragment` is the only place where I was getting/creating the VM before